imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -600,17 +611,17 @@ def process_swapi_science(
     )
 
     dataset["swp_pcem_counts"] = xr.DataArray(
-        np.array(swp_pcem_counts, dtype=np.uint16),
+        np.array(swp_pcem_counts, dtype=np.float32),
```

Comment:
These are already arrays per the above? So I think you can just put the variable itself here and avoid an extra copy.

```suggestion
        swp_pcem_counts,
```